### PR TITLE
Sanitize requests when logging

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: tools/test-proxy/Azure.Sdk.Tools.TestProxy/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```
Comment (scbedd) [inline tools/test-proxy/Azure.Sdk.Tools.TestProxy/Properties/AssemblyInfo.cs:3]: Where did this come from? I'm a bit weirded out it wasn't already there, and I don't understand why it needs to be if it wasn't before.

We don't need this if you swapped that property to `public` right?

Comment (JoshLove-msft) [reply tools/test-proxy/Azure.Sdk.Tools.TestProxy/Properties/AssemblyInfo.cs:3]: I made the Logger property internal again so we do need this.

**Review by scbedd — APPROVED** [review]
Thanks for the PR @JoshLove-msft 

Once you get this green, I'm good with these changes. Hoping our actual signing process for the .exe should be unaffected 👍

**Comment by JoshLove-msft** [comment]
> Once you get this green, I'm good with these changes. Hoping our actual signing process for the .exe should be unaffected 👍

Should be unaffected as I moved it directly into the test directory. That is what was initially caught by my changes - the icon was missing when packing due to the targets file not having the icon path.

**Pull request merged**